storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -1480,9 +1481,18 @@ check_if_param_set(const char *param)
 
 my_bool
 xb_get_one_option(int optid,
-		  const struct my_option *opt __attribute__((unused)),
+		  const struct my_option *opt,
 		  char *argument)
 {
+  param_str << "--" << opt->name;
+  if(argument) {
+    if(strcmp(opt->name, "password")==0) {
```

Comment:
Please also consider hiding `encrypt-key` and `transition-key` option values. With that fixed, this PR looks good.

Reply:
ah... also formatting looks a bit odd. There should be spaces around `==` and between `if` and `(`...